acquisitions/src/main/resources/thunderjet/mod-finance/features/ledger-fiscal-year-rollover-order-with-broken-encumbrance.feature

```diff
@@ -0,0 +1,204 @@
+            "rolloverBudgetValue": "None",
+            "setAllowances": false,
+            "addAvailableTo" : "Available"
+          }
+        ],
+        "encumbrancesRollover": [
+          {
+            "orderType": "One-time",
+            "basedOn": "InitialAmount",
+            "increaseBy": 0
+          }
+        ]
+      }
+    """
+    When method POST
+    Then status 201
+
+
+  Scenario: Wait for rollover to end
+    * configure retry = { count: 10, interval: 500 }
+    Given path 'finance/ledger-rollovers-progress'
+    And param query = 'ledgerRolloverId==' + rolloverId
+    And retry until response.ledgerFiscalYearRolloverProgresses[0].overallRolloverStatus != 'In Progress'
+    When method GET
+    Then status 200
+
+
+  Scenario: Check rollover logs for rolloverId=<rolloverId>
+    Given path 'finance/ledger-rollovers-logs', rolloverId
+    When method GET
+    Then status 200
+    And match response.rolloverStatus == 'Success'
+    And match response.ledgerRolloverType == 'Commit'
+
+
+  Scenario: Check rollover statuses rolloverId=<rolloverId>
+    Given path 'finance/ledger-rollovers-progress'
+    And param query = 'ledgerRolloverId==' + rolloverId
+    When method GET
+    Then status 200
+    And match response.ledgerFiscalYearRolloverProgresses[0].overallRolloverStatus == 'Success'
+
+
+  Scenario: Check new budget after rollover
+    Given path 'finance/budgets'
+    And param query = 'fundId==' + fundId + ' AND fiscalYearId==' + toFiscalYearId
+    And retry until response.totalRecords > 0
+    When method GET
+    Then status 200
+    * def newBudgetId = $.budgets[0].id
+
+    Given path 'finance/budgets', newBudgetId
+    When method GET
+    Then status 200
+    And match response.allocated == 100
+    And match response.available == 50
+    And match response.unavailable == 50
+    And match response.netTransfers == 0
+    And match response.encumbered == 50
+
+
+  Scenario Outline: Check encumbrances and order lines after rollover
+    * def orderId = <orderId>
+    * def poLineId = <poLineId>
+    * def amount = <amount>
+    * def initialAmountEncumbered = <initialAmountEncumbered>
+
+    Given path 'finance-storage/transactions'
+    And param query = 'fromFundId==' + fundId + ' AND fiscalYearId==' + toFiscalYearId + ' AND encumbrance.sourcePurchaseOrderId==' + orderId
+    When method GET
+    Then status 200
+    * def transaction = response.transactions[0]
+    * def encumbranceId = transaction.id
+    And match transaction.amount == amount
+    And match transaction.encumbrance.status == 'Unreleased'
+    And match transaction.encumbrance.amountExpended == 0
+    And match transaction.encumbrance.initialAmountEncumbered == initialAmountEncumbered
+
+    * configure headers = headersAdmin
+    * def orderLineResponse = call getOrderLine { poLineId: #(poLineId) }
+    * def response = orderLineResponse.response
+    And match response.fundDistribution[0].encumbrance == encumbranceId
```

Comment:
can you please add some comment or print statement what is the most important field to take an attention on
The most important here that the rollover finished succesfully and po lines remained to be linked to the same encumbrances: valid and not valid encumbrance id?

Reply:
Thank you for review. Described in the print statement